cypress/e2e/user-flows/scenario_slice_association.cy.js

```diff
@@ -0,0 +1,121 @@
+const random = Date.now();
+
+const customTypeName = `My Test ${random}`;
+const customTypeId = `my_test_${random}`;
+
+const sliceName = `TestSlice${random}`;
+const sliceId = `test_slice${random}`; // generated automatically from the slice name
+const sliceLib = "slices";
+
+describe("I am an existing SM user (Next) and I want to associate a Slice to a CT and review my experience.", () => {
+  before(() => {
+    cy.clearProject();
+  });
+
+  beforeEach(() => {
+    cy.setSliceMachineUserContext({});
+  });
+
+  it("Create a Custom type with multiple fields", () => {
+    cy.createCustomType(customTypeId, customTypeName);
+
+    cy.addFieldToCustomType("UID", "ID Field", "uid");
+    cy.addFieldToCustomType("Key Text", "Key Text Field", "key_text_id");
+    cy.addFieldToCustomType("Rich Text", "Rich Text Field", "rich_text_id");
+    cy.saveCustomTypeModifications();
+
+    cy.reload();
```

Comment:
Do we reload to check the save worked as expected? 
We've had a. bug recently where the server state was updated correctly but the browser state wasn't. I wonder which one we'd want to test more here (also users usually don't reload)

Reply:
Indeed we reload to see if the state persisted. This is what is used in the scenarios.
I guess it makes more sense because if changes are persisted, the app still has a way of running.

I don't know if we'd be able to test the Redux state or if we'd want to do that.

Reply:
I agree that reloading is a good way to make sure the persisted data can be read/understood by the app. 
I don't think the e2e tests should check the Redux state since it's an implementation detail.